0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md

```diff
@@ -10,6 +10,31 @@ Connectors discover their peers through out-of-band communication, or by looking
 
 Once peered, two connectors have a ledger between them; this is often a ledger with just two accounts, often administered collaboratively by the two connectors.
 
+There are two ways for connectors to peer with each other: WebFinger-based, and `ilp_secret`-based.
+
+In WebFinger-based discovery, both peers look up each other's host resource, for instance:
+
+* the server wallet1.com looks up https://wallet2.com/.well-known/webfinger?resource=https://wallet2.com
+* the server wallet2.com looks up https://wallet1.com/.well-known/webfinger?resource=https://wallet1.com
+* out of band, they agree on a currency code and a currency scale for the peering ledger
+
+This way, each peer has the other peer's public key. They now use ECDH to create a shared secret, from which a ledger prefix and an auth token are derived.
+
+In `ilp_secret`-based peering, each peer generates an `ilp_secret` URN and gives that to the other one, to achieve the same:
```

Comment:
I'm not sure I would add this yet. @justmoon made a good point that the `ilp_secret` format is longer and doesn't necessarily provide any benefit over using the plain rpc connection string. We need to discuss whether this is the right format to go with.